src/modules/dashboard/components/ColonyFundingMenu/ColonyFundingMenu.tsx

```diff
@@ -53,7 +53,7 @@ const ColonyFundingMenu = ({
   selectedDomainId,
 }: Props) => {
   const { walletAddress, networkId, ethereal, username } = useLoggedInUser();
-  const isNetworkAllowed = !!ALLOWED_NETWORKS[networkId || 1];
+  const isNetworkAllowed = checkIfNetworkIsAllowed(networkId);
```

Comment:
Very nice refactoring all these files to use your `checkIfNetworkIsAllowed` util